internal/kafka/internal/services/data_plane_kafka.go

```diff
@@ -239,6 +240,7 @@ func (d *dataPlaneKafkaService) setKafkaClusterReady(kafka *dbapi.KafkaRequest)
 
 	if shouldSendMetric {
 		metrics.UpdateKafkaRequestsStatusSinceCreatedMetric(constants2.KafkaRequestStatusReady, kafka.ID, kafka.ClusterID, time.Since(kafka.CreatedAt))
+		metrics.UpdateKafkaRequestsStatusSinceLastSetMetric(constants2.KafkaRequestStatusReady, kafka.ID, kafka.ClusterID)
```

Comment:
@machi1990 @JameelB @pb82 - should I actually move this outside of the `shouldSendMetric` check and only send it, if previous status was resuming?

Reply:
At the moment, with this we won't update the Kafka metric if the previous status is `resuming` and we'll consider that Kafka in resuming state. 

I think the `metrics.UpdateKafkaRequestsStatusSinceLastSetMetric(constants2.KafkaRequestStatusReady, kafka.ID, kafka.ClusterID)` has to be fired when the previous status is either: `provisioning` or `resuming`.